plasmapy/dispersion/numerical/hollweg_.py

```diff
@@ -0,0 +1,271 @@
+        assumes that ion motion has only one degree of freedom, namely
+        along magnetic field lines.
+    z_mean : `float` or int, optional
+        The average ionization state (arithmetic mean) of the ``ion`` composing
+        the plasma.  Will override any charge state defined by argument ``ion``.
+    Returns
+    -------
+    omega : Dict[str, `~astropy.units.Quantity`]
+        A dictionary of computed wave frequencies in units :math:`rad/s`.  The
+        dictionary contains three keys: ``'fast_mode'`` for the fast mode,
+        ``'alfven_mode'`` for the Alfvén mode, and ``'acoustic_mode'`` for the
+        ion-acoustic mode.  The value for each key will be a :math:`N x M` array.
+    Raises
+    ------
+    TypeError
+        If applicable arguments are not instances of `~astropy.units.Quantity` or
+        cannot be converted into one.
+    TypeError
+        If ``ion`` is not of type or convertible to `~plasmapy.particles.Particle`.
+    TypeError
+        If ``gamma_e``, ``gamma_i``, or``z_mean`` are not of type `int` or `float`.
+    ~astropy.units.UnitTypeError
+        If applicable arguments do not have units convertible to the expected
+        units.
+    ValueError
+        If any of ``B``, ``k``, ``n_i``, ``T_e``, or ``T_i`` is negative.
+    ValueError
+        If ``k`` is negative or zero.
+    ValueError
+        If ``ion`` is not of category ion or element.
+    ValueError
+        If ``B``, ``n_i``, ``T_e``, or ``T_I`` are not single valued
+        `astropy.units.Quantity` (i.e. an array).
+    ValueError
+        If ``k`` or ``theta`` are not single valued or a 1-D array.
+
+    Notes
+    -----
+    Solves the equation 3 in Bellan2012JGR (equation 38 in Hollweg1999)
+    .. math::
+        \left(\frac{\omega^2}{k_{\rm z}^2 v_{\rm A}^2} - 1 \right) &
+        \left[\omega^2\left(\omega^2 - k^2 v_{\rm A}^2\right) - &
+        \beta k^2 v_{\rm A}^2 \left(\omega^2 - k_{\rm z}^2 v_{\rm A}^2 \right) &
+        \right]\\ = \omega^2 \left(\omega^2 - k^2 v_{\rm A}^2 \right) k_{\rm x}^2 &
+        \left(\frac{c_{\rm s}^2}{\omega_{\rm ci}^2} - \frac{c^2}{\omega_{\rm pe}^2} &
+        \frac{\omega^2}{k_{\rm z}^2v_{\rm A}^2}\right)
+    where
+    .. math::
+        k_{\rm x} = \mathbf{k} \cdot \hat{x}
+    Examples
+    --------
+    >>> from astropy import units as u
+    >>> from plasmapy.dispersion.numerical import hollweg_
+    >>> inputs = {
+    ...    "k": np.logspace(-7, -2, 2) * u.rad / u.m,
+    ...    "theta": 88 * u.deg,
+    ...    "n_i": 5 * u.cm ** -3,
+    ...    "B": 2.2e-8 * u.T,
+    ...    "T_e": 1.6e6 * u.K,
+    ...    "T_i": 4.0e5 * u.K,
+    ...    "ion": Particle("p+"),
+    ... }
+    >>> omegas = hollweg(**inputs)
+    >>> omegas
+    {'fast_mode': <Quantity [2.62911663e-02, 2.27876968e+03] rad / s>,
+    'alfven_mode': <Quantity [7.48765909e-04, 2.13800404e+03] rad / s>,
+    'acoustic_mode': <Quantity [0.00043295, 0.07358991] rad / s>}"""
```

Comment:
```suggestion
    {'fast_mode': <Quantity [2.62911663e-02, 2.27876968e+03] rad / s>,
     'alfven_mode': <Quantity [7.48765909e-04, 2.13800404e+03] rad / s>,
     'acoustic_mode': <Quantity [0.00043295, 0.07358991] rad / s>}
    """
```

I believe this will get the "Linter" check to pass.

Reply:
I think this was one of the things setting the test off. There were some other spacing issues that it didn't like but the test passes now.